koku/koku/rbac.py

```diff
@@ -193,11 +194,17 @@ def _request_user_access(self, url, headers):
             RBAC_CONNECTION_ERROR_COUNTER.inc()
             raise RbacConnectionError(err)
 
+        if response.status_code >= status.HTTP_500_INTERNAL_SERVER_ERROR:
+            msg = ">=500 Response from RBAC"
+            LOGGER.warning(msg)
+            RBAC_CONNECTION_ERROR_COUNTER.inc()
+            raise RbacConnectionError(msg)
+
         if response.status_code != status.HTTP_200_OK:
             try:
                 error = response.json()
                 LOGGER.error("Error requesting user access: %s", error)
-            except ValueError as res_error:
+            except (JSONDecodeError, ValueError) as res_error:
                 LOGGER.error("Error processing failed, %s, user access: %s", response.status_code, res_error)
```

Comment:
I would think we would change this to a warning to avoid sentry triggers.
```suggestion
                LOGGER.warning("Error processing failed, %s, user access: %s", response.status_code, res_error)
```